pontoon/base/models.py

```diff
@@ -180,6 +183,28 @@ def user_translated_locales(self):
 
     return [locale.code for locale in locales]
 
+
+@property
+def user_rights(self):
+    output = 'Contributor'
+
+    if self.translated_locales:
+        if self.is_superuser:
+            locales = 'all locales'
```

Comment:
Actually, can we just call the user 'Admin' if it's a super user (without listing any locales)?